.eslintrc.js

```diff
@@ -203,7 +203,7 @@ module.exports = {
 
     '/packages/amplify-graphql-types-generator/test', // Not linting tests yet
     'amplify-e2e-tests', // Not linting test projects yet
-    'amplify-console-integration',
+    'amplify-console-integration-tests',
```

Comment:
Since this is new package we should not add it here in lint ignore. We have e2e tests here because lint was enabled later. Our goal is to make sure everything passes the linter